Update Documentation (#153)
NethermindEth/sedge: docs/docs/intro.md

# Introduction

**Sedge** is a one-click setup tool for PoS network/chain validators and nodes written entirely in the [Go programming language](https://golang.org/). **Sedge** takes care of the entire on-premise full node setup based on the chosen client, using generated docker-compose scripts based on the desired configuration. The following list shows some of the use cases for Sedge:

> - **Staking**. You can use Sedge to set up and deploy an Ethereum and Gnosis full node, including a validator node, and nodes with mev-boost. Sedge's on-premise setup design favors solo stakers but can also be integrated with other staking solutions.

Reviewer: ```suggestion
- **Staking**. You can use Sedge to set up and deploy an Ethereum and Gnosis full node, including a validator node and nodes with mev-boost. Sedge's on-premise setup design favors solo stakers but can also be integrated with other staking solutions.
```

:::

## Why you should try Sedge

> Sedge is a tool that focuses on ease of use and as such lowers the entry barrier for non technical users. With Sedge, you can run a node or a validator on Ethereum with little to no technical background and set it all up in less than 5 minutes! Don't know which client to choose? No problem! Let Sedge choose it for you.

Reviewer: ```suggestion
Sedge is a tool that focuses on ease of use and, as such, lowers the entry barrier for non-technical users. With Sedge, you can run a node or a validator on Ethereum with little to no technical background and set it all up in less than 5 minutes! Don't know which client to choose? No problem! Let Sedge choose it for you.
```

If you have enough technical knowledge, you can also take advantage of Sedge and its features. You can generate the setup files, modify them according to your needs, and integrate Sedge with other solutions.

> Sedge codebase design allow us to create many features in a way Sedge will be able to satisfy most of your needs: setup a single node or a full node, cross platform support, alerting mechanisms, support for other networks and much more. [Stay tuned](https://twitter.com/nethermindeth) to get the most of these features as soon as we ship them.

Reviewer: ```suggestion
The Sedge codebase design allows us to create many features and support you with setting up a single node or a full node, cross-platform support, alerting mechanisms, support for other networks, and much more. [Stay tuned](https://twitter.com/nethermindeth) to find out when they’re shipped!
```
